src/common/CommonUtils.ts

```diff
@@ -49,5 +48,34 @@ export class CommonUtils {
         });
     }
 
+    public static getLwcServerPort(): string | undefined {
+        const getProcessCommand =
+            process.platform === 'win32'
+                ? 'wmic process where "CommandLine Like \'%force:lightning:lwc:start%\'" get CommandLine  | findstr "sfdx.js"'
+                : "ps -ax | grep 'force:lightning:lwc:start' | grep 'sfdx.js' | grep -v grep";
+
+        try {
+            const result = CommonUtils.executeCommand(getProcessCommand).trim();
+            // The result of the above command would be in the form of [ "........./sfdx.js" "force:lightning:lwc:start" ]
+            // when no port is specified, or in the form of [ "........./sfdx.js" "force:lightning:lwc:start" "-p" "1234" ]
+            // when a port is specified.
+
+            let port = '3333'; // default to 3333
+            const pIndex = result.indexOf('-p');
+            if (pIndex > 0) {
+                port = result
+                    .substr(pIndex + 2)
+                    .replace(/"/gi, '')
+                    .trim();
+            }
+            return port;
+        } catch {
+            // If we got here it's b/c the grep command fails on empty set,
+            // which means that the server is not running
+        }
+
+        return undefined;
```

Comment:
Can we move this inside the `catch` block?

Reply:
sure